src/WebJobs.Script.WebHost/Standby/StandbyManager.cs

```diff
@@ -182,7 +183,14 @@ public async Task InitializeAsync()
 
         private async Task CreateStandbyWarmupFunctions()
         {
-            string scriptPath = _options.CurrentValue.ScriptPath;
+            ScriptApplicationHostOptions options = _options.CurrentValue;
+
+            if (!options.IsStandbyConfiguration)
+            {
```

Comment:
Can you add a log here with a queryable EventId? Similar to what I did in #5375. It'd be good to be able to query how often this happens.

Reply:
Done

Reply:
Done.